assets/scss/00-base/_breakpoints.scss

```diff
@@ -30,3 +30,6 @@ $bp-centered-content-width: "min-width: 1315px";
 // switch the header from full width to hamburger
 $bp-header-toggle-min: "min-width: 841px";
 $bp-header-toggle-max: "max-width: 840px";
+
+// Feedback form custom breakpoint 
+$bp-feedback-min: "min-width: 650px";

```

Comment:
Is this just for modal?  If so, could it be removed since we are no longer doing modal?